spec/controllers/timelogs_controller_spec.rb

```diff
@@ -75,39 +75,39 @@
     it "updates your timelog with valid data" do
       timelog = FactoryGirl.create(:timelog, user_id: user.id)
       other_timelog = FactoryGirl.create(:timelog, user_id: user.id)
-      patch :update, params: { id: user.id, format: timelog.id, timelog: other_timelog.attributes }
+      patch :update, params: { id: timelog.id, timelog: other_timelog.attributes }
       timelog.reload
       expect(timelog.start_time).to eql(other_timelog.start_time)
     end
 
     it "admin can update any timelog with valid data" do
       timelog = FactoryGirl.create(:timelog, user_id: user.id)
       other_timelog = FactoryGirl.create(:timelog, user_id: user.id)
-      patch :update, params: { id: admin.id, format: timelog.id, timelog: other_timelog.attributes }
+      patch :update, params: { user_id: admin.id, id: timelog.id, timelog: other_timelog.attributes }
       timelog.reload
       expect(timelog.start_time).to eql(other_timelog.start_time)
     end
 
     it "won't update not your timelog with valid data" do
       timelog = FactoryGirl.create(:timelog, user_id: user.id)
       other_timelog = FactoryGirl.create(:timelog, user_id: other_user.id)
-      patch :update, params: { id: user.id, format: other_timelog.id, timelog: timelog.attributes }
-      other_timelog.reload
-      expect(other_timelog).to eql(other_timelog)
+      expect {
+        patch :update, params: { id: other_timelog.id, timelog: timelog.attributes }
+        }.to raise_error(ActiveRecord::RecordNotFound)      
     end
 
     it "won't update your timelog with bad data" do
       timelog = FactoryGirl.create(:timelog, user_id: user.id)
       bad_timelog = FactoryGirl.build(:timelog, user_id: user.id, start_time: "text")
-      patch :update, params: { id: user.id, format: timelog.id, timelog: bad_timelog.attributes }
+      patch :update, params: { id: timelog.id, timelog: bad_timelog.attributes }
       timelog.reload
       expect(timelog.start_time).to_not eql(bad_timelog.start_time)
     end
 
     it "admin won't update timelog with bad data" do
       timelog = FactoryGirl.create(:timelog, user_id: user.id)
       bad_timelog = FactoryGirl.build(:timelog, user_id: user.id, start_time: "text")
-      patch :update, params: { id: admin.id, format: timelog.id, timelog: bad_timelog.attributes }
+      patch :update, params: { user_id: admin.id, id: timelog.id, timelog: bad_timelog.attributes }
       timelog.reload
       expect(timelog.start_time).to_not eql(bad_timelog.start_time)
```

Comment:
I saw you changing policy, but i don't see you changing policy tests. If they were green before, then they are not fully covering the functionality.